components/Inspector.scss

```diff
@@ -2,8 +2,9 @@
   display: flex;
   flex: 0 0;
   flex-direction: column;
-  transition: flex-basis 0.5s;
-  box-shadow: 0px 7px 3px var(--inset-shadow-color, rgba(214, 214, 214, 0.467));
+  transition: flex-basis var(--transition-duration);
+  border-top-left-radius: var(--rad_lg);
+  border-bottom-left-radius: var(--rad_lg);
```

Comment:
I think https://github.com/npmgraph/npmgraph/pull/224 made it a little too _flat_ so restoring some rounded corners and moving the shadow back inside livened it up a bit:

<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td><img width="442" alt="Screenshot 12" src="https://github.com/user-attachments/assets/2fbab3c7-0375-41ea-a1a7-28daf1a9ff19">
	<td><img width="442" alt="Screenshot 11" src="https://github.com/user-attachments/assets/d4be24ef-a7c6-4eeb-be46-b3ab69eafe04">
</table>